oic.r.cred.raml

```diff
@@ -215,10 +201,6 @@ traits:
               {
                 "credid": 55,
                 "subjectuuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9",
-                "subject": {
-                  "idt": "0",
-                  "id": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9"
-                },
                 "roleid": {
                   "authority": "484b8a51-cb23-46c0-a5f1-b4aebef50ebe",
                   "roles": [
```

Comment:
As per CR36, roleid is of type oic.sec.roletype. oic.sec.roletype has two fields, both are string. It is {"authority": "", "role":""} (note it is role and not roles)